x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/parser/EsqlParser.java

```diff
@@ -50,11 +60,14 @@ private <T> T invokeParser(
         BiFunction<AstBuilder, ParserRuleContext, T> result
     ) {
         try {
-            EsqlBaseLexer lexer = new EsqlBaseLexer(new CaseChangingCharStream(CharStreams.fromString(query)));
```

Comment:
No need to wrap the inputstream to `CaseChangingCharStream` - ANTLR already handles the case insensitivity for us.